scripts/dream.py

```diff
@@ -320,6 +347,14 @@ def create_argv_parser():
         default='cuda',
         help='Device to run Stable Diffusion on. Defaults to cuda `torch.cuda.current_device()` if avalible',
     )
+    parser.add_argument(
+        '--prompt_as_dir',
+        '-p',
+        type=str,
+        nargs='?',
+        const='y',
+        help='Output images are placed in subdirectories named using the prompt string. Only used if --prompt_as_dir or -p is specified.\nIf set to \"simple\" then all non-alphanumeric characters in the prompt will be replaced by underscores for the folder name.',
```

Comment:
I'm not sure what "Only used if --prompt_as_dir or -p is specified" is trying to say here? All flags are only used if specified; that's how flags work.

Reply:
Was copied over from the previous PR, should probably be dropped yeah.